src/jvmMain/kotlin/com/sdercolin/vlabeler/video/FindVideo.kt

```diff
@@ -0,0 +1,28 @@
+package com.sdercolin.vlabeler.video
+
+import java.io.File
+import java.io.FileNotFoundException
+
+sealed interface VideoFinder {
```

Comment:
If you use `sealed interface` or `sealed class`, `enum` is not needed (they are similar so we choose any of it, not both)
```kotlin

sealed interface FindVideoStrategy {
    fun find(path: String, acceptableExtensions: List<String>): String

    object SamePlaceOfReferenceAudio : FindVideoStrategy {
        override fun find(path: String, acceptableExtensions: List<String>): String {
            val fileName = path.substringBeforeLast('.')
            return acceptableExtensions.asSequence().map { fileName + it }.filter { File(it).isFile }.firstOrNull() ?: (
                throw FileNotFoundException(
                    "Video not found at the same place of audio ($path)" +
                        "with name $fileName and extensions among $acceptableExtensions",
                )
                )
        }
    }

    object UserSpecifyLocation : FindVideoStrategy {
        override fun find(path: String, acceptableExtensions: List<String>): String {
            TODO("Not yet implemented")
        }
    }
}
```

Reply:
`UserSpecifyLocation` can be removed if it's not used currently. We don't keep unused code, instead you can add a comment like
``` kotlin
// TODO: implement other strategies such as `UserSpecifyLocation`
```

Reply:
Done.